### DISCUSSION TITLE
UI logs path

### DISCUSSION THREAD

**Discussion by aaHorse** [apache/skywalking#7936] [Q&A]
How do I configure the UI log path?

**Comment by wu-sheng** [comment]
What do you mean log path?

**Comment by aaHorse** [comment]
I want to change the log printing path of the UI, but can't find the configuration method.Only the configuration path of the OAP is described here.
![image](https://user-images.githubusercontent.com/38253852/137481507-e5302e94-a996-4bba-bc6c-b1b8fbe68c81.png)

**Reply by wu-sheng** [reply]
Still confused about `log printing path of the UI`. Webapp is the backend application, and bases on Spring Gateway container.
